### PR TITLE
Total current

### PR DESCRIPTION
**Pull request by quielb**
Adds access to the instant_total_curent meter via get function

### CODE REVIEW

**Comment by quielb** [comment]
Resolves #24

File: tesla_powerwall/responses.py [hunk 1]
```diff
@@ -69,6 +69,13 @@ def energy_imported(self):
     def get_energy_imported(self, precision=DEFAULT_KW_ROUND_PERSICION) -> float:
         return convert_to_kw(self.energy_imported, precision)
 
+    @property
+    def instant_total_current(self):
+        return self.assert_attribute("instant_total_current")
+
+    def get_instant_total_current(self, precision=DEFAULT_KW_ROUND_PERSICION) -> float:
+        return round(self.assert_attribute("instant_total_current"), precision)
```
Comment (jrester) [inline tesla_powerwall/responses.py:77]: Could you replace the `assert_attribute` with the direct access to the property to make it conform with the other methods?

**Review by jrester — CHANGES_REQUESTED** [review]
Thanks for submitting! Just one change I would suggest ;)

**Review by jrester — APPROVED** [review]

**Comment by jrester** [comment]
Great, Thanks!

**Pull request merged**